common/on_action/wc_aging_penalties.txt

```diff
@@ -0,0 +1,165 @@
+﻿on_birthday = {
+	on_actions = {
+		on_birthday_aging
+	}
+}
+on_birthday_aging = {
+	trigger = {
+		30 <= { value = age divide = racial_age_multiplier_value }
+	}
+
+	effect = {
+		if = {
+			limit = {
+				has_trait_with_flag = can_not_get_sick
```

Comment:
I would add a new trait flag with its own description. This way you won't need the trait limits below.